contracts/Vault.vy

```diff
@@ -1736,8 +1753,10 @@ def report(gain: uint256, loss: uint256, _debtPayment: uint256) -> uint256:
     #       the Vault based on the Strategy's debt limit (as well as the Vault's).
     totalAvail: uint256 = gain + debtPayment
     if totalAvail < credit:  # credit surplus, give to Strategy
+        self.totalIdle -= credit - totalAvail
         self.erc20_safe_transfer(self.token.address, msg.sender, credit - totalAvail)
```

Comment:
minor gas improvement is to store in memory result of `totalAvail - credit` to use in next line as well

Reply:
same as above